lib/node_modules/@stdlib/math/base/special/ccis/lib/main.js

```diff
@@ -20,44 +20,78 @@
 
 // MODULES //
 
-var computeCis = require( './cis.js' );
+var sincos = require( '@stdlib/math/base/special/sincos' );
```

Comment:
Use `assign` API. In the parent scope, allocate a two-element array buffer `[ 0.0, 0.0 ]`.